<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About PACE4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/pace.png</normaloff>:/icons/pace.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 225);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4,1,1,3,2,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/pace.qrc">:/pace4.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
margin-bottom:0;</string>
       </property>
       <property name="text">
        <string>PACE4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons/pace.qrc">:/pace4.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
padding:0px;
margin: 0px;</string>
       </property>
       <property name="text">
        <string>fusion-evaporation code</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_14">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>125</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(222, 226, 197); background-color: rgb(0, 90, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;JULIAN modified version&lt;br/&gt;Monte-Carlo code coupling angular momentum&lt;br/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;by M.Hillman and Y.Eyal &lt;/span&gt;&lt;/p&gt;&lt;p&gt;PACE : Projection Angular-momentum Coupled Evaporation&lt;br/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;by Avigdor Gavron&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>8</number>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Version">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 255, 200);</string>
     </property>
     <property name="text">
      <string>Version 1.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Date">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(240, 255, 200);</string>
     </property>
     <property name="text">
      <string>28-April-2015</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="label_mail">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>send mail to:</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 5px;
color: rgb(210, 210, 210);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The PACE2 program has been ported to MS Windows by O.B. Tarasov within the framework of the program LISE&lt;span style=&quot; vertical-align:super;&quot;&gt;++&lt;/span&gt;, and named PACE4. The code has been updated with AME database, Q.-M. tunneling approach, utilities, plots.This program has been made into a cross-platform application in the Qt-framework by M.P. Kuchera.&lt;/p&gt;&lt;p&gt;The GUI-version is currently maintained by O.B.Tarasov&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 90, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,3,1">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="2" column="1">
       <widget class="ClickableLabel" name="label_LISE">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;link to LISE++&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(222, 226, 197);</string>
        </property>
        <property name="text">
         <string>http://lise.nscl.msu.edu</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="ClickableLabel" name="label_Charge">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Link to PACE4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(222, 226, 197);</string>
        </property>
        <property name="text">
         <string>http://lise.nscl.msu.edu/pace4.html</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>PACE4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>LISE&lt;sup&gt;++&lt;/sup&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons/pace.qrc">:/emblem_little.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(210, 210, 210);</string>
       </property>
       <property name="text">
        <string>for citations please&lt;br&gt; use the following</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">color: rgb(210, 210, 210);</string>
       </property>
       <property name="text">
        <string>O.B.Tarasov,  D.Bazin, NIM B (2008) 4657;
 A.Gavron, Phys.Rev. C21 (1980) 230-236</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>w_Stuff/w_Label_clickable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/pace.qrc"/>
 </resources>
 <connections/>
</ui>
